python/spec/dependabot/python/update_checker/pipenv_version_resolver_spec.rb

```diff
@@ -255,7 +255,7 @@
                 to start_with("Dependabot detected the following Python")
               expect(error.message).to include("3.4.*")
               expect(error.message).
-                to include("supported in Dependabot: 3.9.0, 3.8.6, 3.8.5")
+                to include("supported in Dependabot: 3.9.1, 3.8.6, 3.8.5")
```

Comment:
```suggestion
                to include("supported in Dependabot: 3.9.1, 3.9.0, 3.8.6")
```

Reply:
I wish we could change this test so we don't have to update it every time we add a new python 🤔